test/format/format_config.c

```diff
@@ -496,6 +496,19 @@ config_run(void)
           REALLOC_MAX_TABLES);
     }
 
+    if (GV(RUNS_PREDICTABLE_REPLAY)) {
+        /*
+         * Predictable replays can get extremely slow with throttling.
+         *
+         * TODO: Investigate reasons for that.
+         */
+        if (GV(OPS_THROTTLE)) {
+            if (config_explicit(NULL, "ops.throttle"))
```

Comment:
Should we check that it is explicit **and** on? It doesnt really matter in the end but i believe we would see that warning message even though we explicitly disabled throttling?

Reply:
@etienneptl I believe `GV(OPS_THROTTLE)` in the line before will only be true when throttling is on.

Reply:
That's right, thank you, @ddanderson!